Dockerfile

```
@@ -24,6 +24,7 @@ RUN addgroup -S lighttpd -g ${GID} && adduser -D -S -u ${UID} lighttpd lighttpd
 WORKDIR /www
 
 COPY lighttpd.conf /lighttpd.conf
+COPY ipv6.sh /etc/lighttpd/ipv6.sh
```

could you rename the `ipv6.sh` script to `lighttpd-ipv6.sh`, to make it clear it's a lighthttp config file at the root of the repo ? thx!
```suggestion
COPY lighttpd-ipv6.sh /etc/lighttpd/ipv6.sh
```

---

Done